### PR TITLE
Template and utils func to create an invoice PDF

### LINKED ISSUE #5374: Email is not completely translated
The main part of the email("You're receiving this e-mail...") is not translated. The parts after are properly translated. However, the text in text/plain is completely correctly translated. I've tried it with different languages (german and russian).

However, the text in text/plain is completely correctly translated:
```
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit



Hi!

Sie bekommen diese E-Mail weil Sie oder jemand anders ein Passwort für Ihren Benutzeraccount auf localhost:8000 angefordert hat.
Sie können diese Nachricht ignorieren, wenn sie es nicht selber angefordert haben.
Klicken Sie unten um Ihr Passwort zurückzusetzen.


http://localhost:8000/de/account/password/reset/MQ/5ez-e13a9786c6c548232f8b/


Dies ist eine automatisch generierte E-Mail, bitte antworten Sie nicht auf diese E-Mail-Adresse.
Mit freundlichen Grüßen, Saleor e-commerce
```

### Steps to reproduce the problem
1.  go to password reset page (e.g. /de/account/password/reset/)
2. provide email and click on send

### What I expected to happen
a fully translated email

### Screenshots
email

**System information**
Operating system: macOS Catalina & Windows 10

### PR DESCRIPTION
**Pull request by tomaszszymanski129**
I want to merge this change because it contains the new PDF template and helper funcs to generate an invoice.

Also resolves #5374 as it updates translations (I also pushed to transifex)

Sample invoice: 
[0013e222-f730-4a30-aaa3-65133ae9ca31.pdf](https://github.com/mirumee/saleor/files/4515887/0013e222-f730-4a30-aaa3-65133ae9ca31.pdf)

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: templates/invoice.html [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+            <br />
+            <span class="normal-text">{{ order.billing_address.phone }}</span>
+            <br />
+            <span class="normal-text">{{ order.get_customer_email }}</span>
+            <br />
+        </div>
+    </div>
+    <div class="section-invoice-info section-right">
+        <div class="content-padded">
+            <span class="header-title">INVOICE INFORMATION</span>
```
Comment (maarcingebala) [inline templates/invoice.html:155]: As long as we keep templates in the core repo, we should wrap these string with `trans` tags (with context).

File: templates/invoice.html [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+            float: left;
+        }
+
+        .section-right {
+            width: 50%;
+            float: right;
+        }
+
+        .header-category {
+            font-family: Inter;
```
Comment (maarcingebala) [inline templates/invoice.html:29]: According to the design, we should also use this font for the body of the invoice.

File: templates/invoice.html [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+            color: rgba(40, 35, 74, 0.6);
+
+        }
+
+        .header-item {
+            display: block;
+            color: black;
+            font-weight: bold;
+
+            font-family: Inter;
```
Comment (NyanKiyoshi) [inline templates/invoice.html:43]: Are you sure it loaded? I see a serif font. Also, it is always good practice to set a fallback font, sans-serif here

File: saleor/order/utils.py [hunk 4]
```diff
@@ -338,3 +343,18 @@ def get_voucher_discount_for_order(order: Order) -> Money:
 
 def match_orders_with_new_user(user: User) -> None:
     Order.objects.confirmed().filter(user_email=user.email, user=None).update(user=user)
+
+
+def _get_unique_pdf_file_name():
+    while True:
+        pdf_path = f"{settings.MEDIA_ROOT}/{token_urlsafe(32)}.pdf"
+        if not os.path.isfile(pdf_path):
+            return pdf_path
```
Comment (NyanKiyoshi) [inline saleor/order/utils.py:352]: Hmmm what about UUID4 rather than this O(N) loop?

File: templates/invoice.html (hunk 3 above)
Comment (tomaszszymanski129) [reply templates/invoice.html:43]: It look like so but I will check again, weasyprint ignores a lot of stuff unfortunately :/

File: saleor/order/utils.py (hunk 4 above)
Comment (NyanKiyoshi) [reply saleor/order/utils.py:352]: This will also fail if we decide not to use local media storage, i.e. an instance storage gets wiped

Comment (NyanKiyoshi) [inline saleor/order/utils.py:352]: What about people that don't want local media storage, i.e. scaling? I would definitely not want this part making system calls onto my application

**Comment by NyanKiyoshi** [comment]
Once again, prefer UUID4, trust it; race condition is VERY unlikely in that context. Do not make system calls, they are heavy and totally unneeded.

File: saleor/order/utils.py (hunk 4 above)
Comment (maarcingebala) [reply saleor/order/utils.py:352]: Yeah I guess we could just UUID as we do for example for checkout/order tokens.

File: templates/invoice.html (hunk 3 above)
Comment (NyanKiyoshi) [reply templates/invoice.html:43]: Check the logs, it tells a lot

File: saleor/order/utils.py [hunk 5]
```diff
@@ -338,3 +342,13 @@ def get_voucher_discount_for_order(order: Order) -> Money:
...
+
+
+def generate_invoice_pdf_for_order(invoice):
+    logo_path = static_finders.find("images/logo-light.svg")
+    rendered_template = get_template("invoice.html").render(
+        {"invoice": invoice, "order": invoice.order, "logo_path": f"file://{logo_path}"}
+    )
+    return HTML(string=rendered_template).write_pdf(
+        f"{settings.MEDIA_ROOT}/{uuid4()}.pdf"
+    )
```
Comment (NyanKiyoshi) [inline saleor/order/utils.py:354]: What about people using custom media storage?

File: templates/invoice.html (hunk 3 above)
Comment (tomaszszymanski129) [reply templates/invoice.html]: It is loaded properly, also added fallback font as suggested (in body tag), logs don't always say something is ignored, weasyprint is not a full rendering engine therefore some stuff is simply not working as expected. But managed to make it look all right.

File: templates/invoice.html [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+        </div>
+    </div>
+    <div class="section-invoice-info section-right">
+        <div class="content-padded">
+            <span class="header-title">
+                {% trans "INVOICE INFORMATION" context "Invoice PDF header" %}
+            </span>
+            <br />
+            <span class="header-category">
+                {% trans "Order:" context "Invoice PDF header" %}
```
Comment (NyanKiyoshi) [inline templates/invoice.html:162]:
```suggestion
                {% trans "Order:" context "Invoice PDF header: order number" %}
```

Helpful for translating

File: templates/invoice.html [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+        </div>
+    </div>
+    <div class="content-tight-padded">
+        <div class="content-separator">&nbsp;</div>
+    </div>
+    <div class="section-left">
+        <div class="content-padded">
+            <span class="header-title">{% trans "PAYMENT METHOD" context "Invoice PDF" %}</span>
+            <br />
+            <span class="normal-text">Credit Card</span>
```
Comment (NyanKiyoshi) [inline templates/invoice.html:247]: Placeholder?

Comment (tomaszszymanski129) [reply templates/invoice.html:247]: Whoops! Yep, wrrr I knew I was about to change something just forgot what it was.

**Comment by pwgryglak** [comment]
@maarcingebala @tomaszszymanski129.
Design-wise we need to:
- Improve spacings for headers
- Improve line heights for texts
- Align prices to right - also for table headers for prices.
- What font are we using in the invoices? It seems like some Monospace in some places, and some sans-serif in others. We need to keep it consistent with one font.

**Comment by tomaszszymanski129** [comment]
@pwgryglak the pdf attached is sort of WIP (spacing), Im working on multi page PDF now, will adjust these things there. About font this is 'Inter' as I copied that from design, should it be changed?

**Comment by tomaszszymanski129** [comment]
going to merge that since that is to feature branch anyway. Will handle multi-page and design adjustments on subsequent PR.

**Pull request merged**